### PR TITLE
add interface to get the projection/ellps/prime_meridian/units lists

### PR DESCRIPTION
**Pull request by snowman2**
Addresses #240

### CODE REVIEW

File: test/test_list.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    assert isinstance(ang_map["deg"], Unit)
+    assert ang_map["deg"].id == "deg"
+    assert ang_map["deg"].name == "Degree"
+
+
+def test_get_ellps_map():
+    ellps_map = get_ellps_map()
+    assert ellps_map["WGS84"]["description"] == "WGS 84"
+    assert_almost_equal(ellps_map["WGS84"]["a"], 6378137.0, decimal=1)
+    assert_almost_equal(ellps_map["WGS84"]["rf"], 298.257223563, decimal=1)
```
Comment (micahcochran) [inline test/test_list.py]: The "rf" factor should be tested to a few more decimal places.  Several of them have "rf" factors around 298.25.

Comment (snowman2) [reply test/test_list.py]: This is mostly just checking that it is in the correct format. It does not attempt to check values as that should be handled by PROJ.

**Comment by snowman2** [comment]
I updated `Geod.__init__()` as well.

**Comment by micahcochran** [comment]
Thanks @snowman2

**Pull request merged**